guides/doc-Upgrading_Disconnected_Project/master.adoc

```diff
@@ -0,0 +1,60 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:upgrading-disconnected:
+:context: upgrading-disconnected
+:mode: disconnected
+
+= {UpgradingDisconnectedDocTitle}
+
+// This guide is not ready for Debian
+ifdef::foreman-deb[]
+include::common/modules/snip_guide-not-ready.adoc[]
+endif::[]
+ifndef::foreman-deb[]
```

Comment:
Anyway, I think that the same policy should apply as for Installing Disconnected.

Reply:
Works for me. If we want to include this in downstream, I will open a follow-up PR. I think running Foreman in a disconnected environment is not officially supported by the Foreman Community.

Reply:
It should work, but we don't have instructions to sync the right repositories so the manual wouldn't work. Because of that, for now it should be satellite only. Please include that description as a comment.